android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -126,6 +130,15 @@ public void setAutoRefresh(final boolean enabled)
         }
     }
 
+    // Called after the all AdView properties are set
+    public void onSetProps()
+    {
+        if ( shouldAddViewUpdate.compareAndSet( true, false ) )
```

Comment:
Generally - we want to avoid adding more state to our code as much as possible. Is `shouldAddViewUpdate` `true` only when `adFormat` and `adUnitId` are set? If so, can we just use that as the conditional?

Reply:
Have you seen use cases where publishers use the same `AppLovinMAX.AdView` widget but for different Ad Unit IDs or ad formats?

We (when BD onboards new pubs) discourage any publishers from integrating that way. They should only have one Ad Unit ID per ad format.

If we detect this integration case we should log an error and ignore.

Reply:
Added the guard not to create for NativeAdView.   AdView has its guard already.